src/components/Tokens/TokenTable/SearchBar.tsx

```diff
@@ -58,15 +61,17 @@ const SearchInput = styled.input`
   }
 
   @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
-    width: 100%;
+    width: ${({ isExplore, isClicked }) => (isExplore ? (isClicked ? 'min(100%, 200px)' : '0') : '100%')};
   }
 `
 
-export default function SearchBar() {
+export default function SearchBar({ tab }: { tab?: string }) {
   const currentString = useAtomValue(filterStringAtom)
   const [localFilterString, setLocalFilterString] = useState(currentString)
   const setFilterString = useUpdateAtom(filterStringAtom)
   const debouncedLocalFilterString = useDebounce(localFilterString, 300)
+  const isExplore = useInfoExplorePageEnabled()
+  const [isClicked, setIsClicked] = useState(false)
```

Comment:
rename to `isOpen` since you want it to remain open when a user has entered input.